addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/ReceiptScreen.xml

```diff
@@ -6,27 +6,53 @@
+                        <h1 t-if="orderUiState.view === 'default'">
+                            <span><t t-esc="orderAmountPlusTip" /></span>
                         </h1>
                     </div>
-                    <div class="button next" t-att-class="{ highlight: !locked }"
-                          t-on-click="orderDone">
-                        Next Order <i class="fa fa-angle-double-right"></i>
+                    <div t-if="orderUiState.view !== 'default'" class="button next"
+                        t-att-class="{ highlight: !locked }" t-on-click="orderDone">
+                        New Order <i class="fa fa-angle-double-right"></i>
                     </div>
                 </div>
-                <div class="centered-content">
-                    <div t-if="state.printInvoiceIsShown" class="button print_invoice"
-                         t-on-click="onPrintInvoice">
-                        <i class="fa fa-print"></i> Print Invoice
-                    </div>
-                    <div class="button print" t-on-click="printReceipt">
-                        <i class="fa fa-print"></i> Print Receipt
-                    </div>
+                <div t-if="orderUiState.view === 'default'" class="default-view">
                     <div class="pos-receipt-container">
                         <OrderReceipt order="currentOrder" t-ref="order-receipt" />
                     </div>
+                    <div class="actions">
+                        <h1>How would you like to receive your receipt?</h1>
+                        <div class="buttons">
+                            <div class="button print" t-on-click="printReceipt">
+                                <i class="fa fa-print"></i> Print Receipt
+                            </div>
+                            <div class="button close" t-on-click="onClose">
+                                <i class="fa fa-times"></i> Close
+                            </div>
+                        </div>
+                        <form t-on-submit.prevent="onSendEmail" class="send-email">
+                            <div class="email-icon"><i class="fa fa-envelope-o" aria-hidden="true"></i></div>
+                            <div class="input-email">
+                                <input type="email" placeholder="Email Receipt" t-model="orderUiState.inputEmail" />
+                                <button class="send" t-att-class="{ highlight: is_email(orderUiState.inputEmail) }" type="submit">Send</button>
+                            </div>
+                        </form>
+                        <t t-if="orderUiState.emailFailed">
+                            <div class="notice failed">
+                                <t t-esc="orderUiState.emailFailedMessage"></t>
+                            </div>
+                        </t>
+                    </div>
+                </div>
+                <div class="done-view" t-else="">
```

Comment:
@caburj I am not sure we need this done view, it create one more step before going to the next order. Do you think we could remove it? as we already get errors if something went wrong.

Did you try this nex screen with the IOT box, and the option automatically printing, to be sure that erverything is oK?

Thanks

Reply:
It indeed add one more step. It is in the mockup so I added it. I haven't tried printing with iot box.
- I'll remove the done-view as I agree with it.
- I'll test the printing with IoT box.

Reply:
cool, otherwise, ti seems really good